app/Http/Controllers/Dashboard/SettingsController.php

```diff
@@ -34,6 +34,22 @@ class SettingsController extends Controller
      */
     protected $subMenu = [];
 
+
+    /**
+     * Array of mail drivers.
+     *
+     * @var string[]
+     */
+    protected $mailDrivers = [
```

Comment:
We should move this into a View Composer as we re-use the same array in the setup.